src/libsync/owncloudpropagator.cpp

```diff
@@ -270,7 +270,13 @@ PropagateItemJob* OwncloudPropagator::createJob(const SyncFileItemPtr &item) {
                 job->setDeleteExistingFolder(deleteExisting);
                 return job;
             } else {
-                auto job = new PropagateUploadFile(this, item);
+                static const bool isNg  = !qgetenv("OWNCLOUD_CHUNK_NG").isEmpty(); // FIXME! use server version
```

Comment:
I think it should be rather decided by a server capability. @DeepDiver1975 does the new chunking already add something to the capabilities?